src/components/commons/LoginHeader.tsx

```diff
@@ -177,7 +177,7 @@ export default function LoginHeader() {
         checkAuthTokenAndContinue(() => {
           dispatch(
             setLoggedUser({
-              id: userInfo.userId,
+              id: `${userInfo.name || ""}${userInfo.familyName || ""}`,
```

Comment:
`name` and `familyName` are [mandatory fields](https://github.com/pagopa/pagopa-infra/blob/5ac8441e3f28f91d975869a60e643b6025f45fba/src/domains/checkout-app/api/checkout/checkout_auth_service/v1/_openapi.json.tpl#L349-L352) in GET user response, so the or fallback with empty string can be removed
```suggestion
              id: `${userInfo.name}${userInfo.familyName}`,
```

Reply:
Done in [5a26b76](https://github.com/pagopa/pagopa-checkout-fe/pull/472/commits/5a26b7604051da4b84b3093396a1a14bf603f765)